docs/pages/repo/docs/core-concepts/monorepos/configuring-workspaces.mdx

```diff
@@ -0,0 +1,181 @@
+---
+title: Configuring Workspaces
+description: Turborepo lets you configure each workspace
+---
+
+import Callout from "../../../../../components/Callout";
+
+# Configuring Workspaces
+
+Most monorepos can declare a `turbo.json` in the root directory with a uniform
+set of tasks that apply to all workspaces. But sometimes, a monorepo can contain
+workspaces that need to configure their tasks differently. To accommodate this,
+Turborepo enables you to extend the root configuration with a `turbo.json` in
+any workspace. This flexibility enables a more diverse set of apps and packages
+to co-exist, and allows workspace owners to maintain specialized tasks and
+configuration without affecting other inhabitants of the monorepo.
+
+## How it Works
+
+To override any of the configuration for a task, add a `turbo.json` file in any
+workspace of your monorepo with a top level `extends` key:
+
+```jsonc filename="turbo.json"
+{
+  "extends": ["//"],
+  "pipeline": {
+    // custom configuration for the workspace here
+  }
+}
+```
+
+<Callout>
+  For now, the only valid value for the `extends` key is `["//"]`. The
+`//` syntax, you may recall, signifies "root", similar to how it's used to
+[declare root tasks][1].
+</Callout>
+
+Extended configuration in a workspace can reset or override any of [the
+configuration for a pipeline task][2]. If you don't include a key, it will inherit
+the value from the extended `turbo.json`.
+
+## Examples
+
+To illustrate, let's look at a couple of use cases.
+
+### Different Frameworks
+
+Let's say your monorepo has a many Next.js apps, and one Sveltekit app. By
+default, both frameworks create their build with the `build` script in their
+respective `package.json`s. With a single `turbo.json` you can configure this
+with:
+
+```jsonc filename="turbo.json"
+{
+  "pipeline": {
+    "build": {
+      "outputs": [".next/**", ".svelte-kit/**"],
+    }
+  }
+}
+```
+
+Notice how both `.next/**` and `.svelte-kit/**` need to be specified as outputs,
+even though though the Next.js apps do not generate this output. With Workspace
+overrides, you can instead add a new configuration at
+`apps/my-svelte-kit-app/turbo.json`, and remove the config from the root
+configuration:
+
+```jsonc filename="apps/my-svelte-kit-app/turbo.json"
+{
+  "extends": ["//"],
+  "pipeline": {
+    "build": {
+      "outputs": [".svelte-kit/**"]
+    }
+  }
+}
+```
+
+```diff filename="turbo.json"
+{
+  "pipeline": {
+    "build": {
+-      "outputs": [".next/**", ".svelte-kit/**"]
++      "outputs": [".next/**"]
+    }
+  }
+}
+```
+
+### Specialized Tasks
```

Comment:
In our running tasks docs we use "specific workspace-tasks" as the term for tasks that only appear in a single workspace. Later in this doc we use "Workspace Specific Task". I think we need to unify how we talk about composable turbo.json compared to workspace#task.

Reply:
Agreed, but this was slightly different. I wanted to show that you can add _new_ tasks to workspaces taht don't exist anywhere else. Workspace-Specific tasks, as of today, are just about specialized configuration of _existing_ tasks.